include/mbedtls/md.h

```diff
@@ -260,8 +260,9 @@ const char *mbedtls_md_get_name( const mbedtls_md_info_t *md_info );
  *
  * \param ctx       The generic message-digest context.
  *
- * \returns         \c 0 on success, #MBEDTLS_ERR_MD_BAD_INPUT_DATA if
- *                  parameter verification fails.
+ * \returns         \c 0 on success.
+ * \returns         #MBEDTLS_ERR_MD_BAD_INPUT_DATA if parameter verification
```

Comment:
Minor suggestion: `if the parameter verification fails` instead of `if parameter verification fails`.

Reply:
I would rather standardize the descriptions of parameter verification failures. Currently we have two formats:
- #MBEDTLS_ERR_MD_BAD_INPUT_DATA if parameter verification fails.
- #MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter failure.
I propose using:
- #MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter verification failure.
as this adheres to the "#error on xxx failure" template, but if we'd like to have less work, we can substitute the "on parameter failure" with the first template version, as it is most common in this file.

Reply:
Fixed